src/main/content/support.html

```diff
@@ -53,8 +53,8 @@ <h2>WebSphere Application Server</h2>
                             <img id="twitter_logo" src="/img/twitter_logo.svg" class="d-block mx-auto" alt="twitter">
                         </figure>
                         <div class="card-body">
-                            <h5 class="card-title">@openlibertyio</h5>
-                            <p class="card-text">News & updates</p>
+                            <h5 class="card-title">@{% t global.openlibertyio %}</h5>
```

Comment:
I would prefer to move the `@` into the message file so the message is all together.

Reply:
when we add to message file it shows below error
atopenlibertyio: @openliberty.io
found character that cannot start any token while scanning for the next token